From a79b6d2f5cc035dfb2aca30c84fae4942fc41abf Mon Sep 17 00:00:00 2001
From: John Shao <johnshao@google.com>
Date: Thu, 24 Feb 2022 22:20:11 +0000
Subject: [PATCH] No longer export CallSubjectDialog

This is most likely not used outside of the app and can be potentially
exploited

Bug: 218341397
Test: Manual
Change-Id: I8c0c2bdddb172aba5a41e3fff0413eb48a5f4455
Merged-In: I8c0c2bdddb172aba5a41e3fff0413eb48a5f4455
(cherry picked from commit eadb0b1cc94deaa238bfdf225a504119a8a24388)
(cherry picked from commit af44f5ac3f3fb660c6646d62c6da9b988d767b36)
Merged-In: I8c0c2bdddb172aba5a41e3fff0413eb48a5f4455
---
 AndroidManifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 32d6644a7..7662a2e32 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -572,7 +572,7 @@
         <activity
             android:name="com.android.contacts.dialog.CallSubjectDialog"
             android:theme="@style/Theme.CallSubjectDialogTheme"
-            android:exported="true"
+            android:exported="false"
             android:windowSoftInputMode="stateVisible|adjustResize">
             <intent-filter>
                 <action android:name="android.intent.action.VIEW"/>
-- 
2.36.0.550.gb090851708-goog

